stdlib/private/StdlibCollectionUnittest/CheckMutableCollectionType.swift.gyb

```diff
@@ -446,8 +446,7 @@ self.test("\(testNamePrefix)._withUnsafeMutableBufferPointerIfSupported()/semant
     var c = makeWrappedCollection(test.collection)
     var result = c._withUnsafeMutableBufferPointerIfSupported {
       (bufferPointer) -> OpaqueValue<Array<OpaqueValue<Int>>> in
-      let value = OpaqueValue(bufferPointer.map(extractValue))
-      return value
+      return OpaqueValue(bufferPointer.map(extractValue))
```

Comment:
N.B. This is still not a _single expression_ closure though, because it has a return statement.

Reply:
@slavapestov These are still considered to be single expression closures.

Reply:
Like everyone else on the team, I am still learning Swift. :)